### ISSUE TITLE
Benny from Fiverr 

### ISSUE THREAD

**Issue by devarsalanali** [hirokiman3/Pinky#1]
Hey hiroki, Its me Benny from Fiverr, who developed pinky project recently with. After delivery my order to you Fiverr disabled my account. since I didn't have your email or contact details that's why I am leaving my message here hope you will get it. And if you don't my here is my email and whatsapp details to continue project. itsaliarsalan@gmail.com +923428045564

**Comment by devarsalanali** [comment]
@hirokiman3 are you there ?

**Comment by hirokiman3** [comment]
Yes, we are in great disbelief that fiverr has been frozen. However, there are very many unnatural things to suspect you.
My email address is below.
hirokiman3@gmail.com
Reason.

**Comment by hirokiman3** [comment]
Can you explain why you were frozen?
I believe that if this is a scam, it is too unnatural.
However, I can only judge it as a scam now.

2023年5月28日(日) 1:38 Arsalan Ali ***@***.***>:

**Comment by hirokiman3** [comment]
The email address you sent this to didn't exist.
itsaliarsalan@gmail.com

**Comment by hirokiman3** [comment]
What you are doing at the moment is not reasonable.

**Comment by hirokiman3** [comment]
If this explanation is not given, I have no choice but to demand that the Government of Pakistan arrest me.

I am waiting for an explanation.
